File: app/src/main/java/com/androweed/muslimsimple/ui/quran/QuranViewModel.java
package com.androweed.muslimsimple.ui.quran;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class QuranViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public QuranViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is notifications fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/androweed/muslimsimple/ui/quran/QuranViewModel.kt
package com.androweed.muslimsimple.ui.quran

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class QuranViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is notifications Fragment"
    }
    val text: LiveData<String> = _text
}